lib/ts/usercontext/userContextWrapper.tsx

```diff
@@ -38,8 +38,8 @@ export default function UserContextWrapper(props: { children: React.ReactNode; u
                  * In this case we return a provider so that the userContext hook can be used by the children
```

Comment:
TODO for me: I will review this file properly later, when everything else is sorted out.